src/server.ts

```diff
@@ -0,0 +1,58 @@
+
+//
+// Just the HTTP glue to the functionality, no implementations.
+// All actual implementations are in scanner.ts
+//
+
+import express, {Application, Request, Response} from "express"
+import {configure, doScan, doExport, getStatus, doZero} from "./scanner"
+import * as store from "./store"
+
+import {Election} from "./types"
+
+// for now, we reset on every start
+store.reset()
+
+export const app : Application = express()
+const port = 3002
+
+app.use(express.json())
+
+app.get("/", (_request : Request, response : Response) => {
```

Comment:
If multiple modules are being used, won't this be a shared endpoint? Is there issue of conflicting endpoints?

Reply:
yeah, this is just for the sake of testing the server manually in a standalone way, without going through the proxy. This makes development way easier. I actually *don't* want this page exposed to the main host.